backend/routes/user.go

```diff
@@ -9,6 +9,7 @@ func UsersRoute() chi.Router {
 	r := chi.NewRouter()
 	r.Get("/", controller.GetAllUsers)
 	r.Get("/{userId}", controller.GetUser)
+	r.Get("/login", controller.GetUserByEmail)
```

Comment:
this should just be a url param im pretty sure @cameronkinsella  thoughts?

Reply:
+1 iirc it's not RESTful for GET to have a body

Reply:
it's not a body. It's a query parameter:
```http://localhost:5001/api/auth/login/?email=...```

Reply:
Oh, my bad. LGTM